lib/queue_classic_plus/base.rb

```diff
@@ -67,7 +67,7 @@ def self.can_enqueue?(method, *args)
                )
              AS x"
 
-        result = QC.default_conn_adapter.execute(q, @queue, method, args.to_json)
+        result = QC.default_conn_adapter.execute(q, @queue, method, JSON.dump(args))
```

Comment:
Can we test this? (e.g. by porting over the double-enqueue spec we looked at earlier)

Reply:
`#to_json` can be redefined by anything (in our case earlier, it was redefined by ActiveSupport), so we'd need to add that as a development dependency... I'm not sure it's worth it? 🤷

Reply:
`activesupport` was added recently together with `active_record` (in https://github.com/rainforestapp/queue_classic_plus/pull/35) to reproduce another Rails-specific problem. But yeah, I'm not saying this _must_ be tested if it ends up being too difficult to set up

Reply:
Not sure how my eyes glazed over that when I looked at the gemspec before replying earlier 🙈 I'll try to add a spec real quick.